db/tables.py

```diff
@@ -0,0 +1,26 @@
+from db import settings, DB
+from sqlalchemy import DateTime, Column, String, schema, Float
+
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+
+class Triggers(Base):
+    __tablename__ = 'Triggers'
+    __table_args__ = {'schema': settings.env}
+
+    id = Column('Id', String, primary_key=True, nullable=False)
+    trigged_at = Column('trigged_at', DateTime)
+    symbol = Column('stock_symbol', String)
+    price = Column('price', Float)
+    transaction_type = Column('transaction_type', String)
+
+
+engine = DB.create_engine()
+
+# create schema if doesn't exist
+if not engine.dialect.has_schema(engine, settings.env):
+    engine.execute(schema.CreateSchema(settings.env))
```

Comment:
What does the `settings.env` do here?

Reply:
Schema is based on env. I was thinking having a test environment and a prod one. What you think?

Reply:
Sounds good

Reply:
There is no default value for env. Would it being None cause an error?

Reply:
yeah it would, but `settings.env` is populated from `.envrc` which should be set to `development`